twake/backend/node/config/default.json

```diff
@@ -30,15 +30,13 @@
     }
   },
   "database": {
-    "type": "mongodb",
+    "type": "cassandra",
```

Comment:
You should not change this, we can override it from env, I will show you how

Reply:
My bad I didn't ant to push it, usually it's gitignored 👍